diff --git a/node_modules/@nuxt/content/dist/module.cjs b/node_modules/@nuxt/content/dist/module.cjs
index c1d15cd..08828a3 100644
--- a/node_modules/@nuxt/content/dist/module.cjs
+++ b/node_modules/@nuxt/content/dist/module.cjs
@@ -2354,7 +2354,7 @@ function resolveSource(source) {
     return defineLocalSource(source2);
   });
 }
-const MAX_SQL_QUERY_SIZE = 1e5;
+const MAX_SQL_QUERY_SIZE = 1e10;
 const SLICE_SIZE = 7e4;
 function generateCollectionInsert(collection, data) {
   const fields = [];
diff --git a/node_modules/@nuxt/content/dist/module.mjs b/node_modules/@nuxt/content/dist/module.mjs
index 1693264..733a700 100644
--- a/node_modules/@nuxt/content/dist/module.mjs
+++ b/node_modules/@nuxt/content/dist/module.mjs
@@ -2338,7 +2338,7 @@ function resolveSource(source) {
     return defineLocalSource(source2);
   });
 }
-const MAX_SQL_QUERY_SIZE = 1e5;
+const MAX_SQL_QUERY_SIZE = 1e10;
 const SLICE_SIZE = 7e4;
 function generateCollectionInsert(collection, data) {
   const fields = [];
